pages/letter/index.html

```diff
@@ -1718,7 +1788,30 @@ <h2 class="c-signatures-title mzp-u-title-sm">Signatories<span>(in order of sign
         <li><cite>Gautam Kamath</cite>, Centre on Regulation in Europe (CERRE)</li>
         <li><cite>Khaled Koubaa</cite>, AT Worthy Technology</li>
         <li><cite>Francisco Anguita</cite>, LIDENBROCK AI</li>
+        <li><cite>Gilles Degottex</cite></li>
         <li><cite>Antoine Jeanjean</cite>, OPT2A</li>
+        <li><cite>Sanjeev Mohindra</cite>, MIT</li>
+        <li><cite>Zeina Abi Assy</cite>, Mozilla</li>
+        <li><cite>Kenrya Rankin</cite>, Mozilla Foundation</li>
+        <li><cite>Michael Green</cite>, Desupervised</li>
+        <li><cite>Gosia Wrzesinska</cite>, Coders Co.</li>
+        <li><cite>Pierre-Yves Gibello</cite>, OW2 open source association</li>
+        <li><cite>Bhavish Pahwa</cite>, Mindtickle</li>
+        <li><cite>Corynne McSherry</cite>, Electronic Frontier Foundation</li>
+        <li><cite>Thomas Breslin</cite>, Lund University</li>
+        <li><cite>Igor Krawczuk</cite>, EPFL</li>
+        <li><cite>Pierre Rasmussen</cite>, Ida Infront</li>
+        <li><cite>Caleb Dillinger</cite>, CAI</li>
+        <li><cite>Darja Smite</cite>, Blekinge Institute of Technology</li>
+        <li><cite>George Shaikovski</cite>, Paige AI</li>
+        <li><cite>John Rickman</cite>, Northwell Health</li>
+        <li><cite>Chinmay Shah</cite>, Arrowhead</li>
+        <li><cite>Anita Delévaux</cite>, Citoyenne</li>
+        <li><cite>Francesco Conti</cite>, University of Bologna</li>
+        <li><cite>Andrew Newton</cite>, SixPivot</li>
+        <li><cite>Ivan Skorokhodov</cite>, Snap Inc.</li>
+        <li><cite>Rob Hudson</cite>, Mozilla</li>
```

Comment:
I didn't intentionally sign it. But that's fine. Hopefully the Terminators of the future don't come back in time and try to kill me.